tuf/ngclient/__init__.py

```diff
@@ -4,12 +4,18 @@
 """TUF client public API
 """
 
+
+# requests_fetcher is public but comes from _internal for now (because
+# sigstore-python 1.0 still uses the module from there). requests_fetcher
+# can be moved out of _internal once sigstore-python 1.0 is not relevant.
```

Comment:
should a similar comment be made in the requests_fetcher module, for people looking at the code directly without seeing context here?

Reply:
Maybe yes